src/less/tooltip/ds6/tooltip-base.less

```diff
@@ -0,0 +1,50 @@
+.icon-btn {
+    background-color: transparent;
+    border: 0 none;
+    font-size: 16px;
+    padding: 7px 6px;
+    vertical-align: middle;
+}
+
+.width-100 {
+    width: 100%;
+}
+
+div.tooltip__body {
+    p {
+        width: 90%;
+    }
+}
+
+.div--link-callout {
+    margin: 0 auto;
+    width: 100%;
+}
+
+.margin-8 {
```

Comment:
Are any of classes below this line being used?

Reply:
Yes, it is used

Reply:
It looks like these aren't actually used in the tooltip styling, just the docs.  If so, we should move these to the docs only.

Reply:
In fact, that applies to all classes in this file, like "width-100".  If it only applies to the docs then it shouldn't be part of the tooltip CSS.